File: src/main/java/com/digsol/turnomed/dto/DoctorDTO.java
package com.digsol.turnomed.dto;

import com.google.gson.JsonObject;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@ApiModel(description = "Doctor")
public class DoctorDTO implements BaseDTO {

    @ApiModelProperty(notes = "Doctor's id")
    private Integer id;

    @ApiModelProperty(notes = "Information")
    private JsonObject info;

    @ApiModelProperty(notes = "Content URL")
    private String contentUrl;

    @ApiModelProperty(notes = "Phone")
    private String phone;

    @ApiModelProperty(notes = "User link")
    private UsersDTO user;
}

File: src/main/kotlin/com/digsol/turnomed/dto/DoctorDTO.kt
package com.digsol.turnomed.dto

import com.google.gson.JsonObject
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty
import lombok.Getter
import lombok.Setter

@Getter
@Setter
@ApiModel(description = "Doctor")
data class DoctorDTO(
    @ApiModelProperty(notes = "Doctor's id")
    var id: Integer? = null,

    @ApiModelProperty(notes = "Information")
    var info: JsonObject? = null,

    @ApiModelProperty(notes = "Content URL")
    var contentUrl: String? = null,

    @ApiModelProperty(notes = "Phone")
    var phone: String? = null,

    @ApiModelProperty(notes = "User link")
    var user: UsersDTO? = null
) : BaseDTO
